__tests__/app.test.js

```diff
@@ -173,3 +172,34 @@ describe("PATCH /api/articles/:article_id", () => {
       })
   });
 })
+
+describe("GET /api/users", () => {
+  test("Status 200", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+  });
+  test("returns an array of objects, each with the property 'username'", () => {
+      return request(app)
+          .get("/api/users")
+          .expect(200)
+      .then((response) => {
+          expect(Array.isArray(response.body)).toBe(true)
+          response.body.forEach((user) => {
```

Comment:
It would be preferable that instead of `response.body` being the array of users - instead it is an object with a key of users, which holds the array